lib/__tests__/disableRanges-integration.test.mjs

```diff
@@ -1,9 +1,7 @@
-'use strict';
-
-const blockNoEmpty = require('../rules/block-no-empty');
-const maxLineLength = require('../rules/max-line-length');
-const selectorCombinatorSpaceBefore = require('../rules/selector-combinator-space-before');
-const stringQuotes = require('../rules/string-quotes');
+import blockNoEmpty from '../rules/block-no-empty/index.js';
+import maxLineLength from '../rules/max-line-length/index';
```

Comment:
I believe extension is required. There probably an ESLint rule for this somewhere. Maybe it's a good idea to enabled this kind of check for `*.mjs`?

```suggestion
import maxLineLength from '../rules/max-line-length/index.js';
```

Reply:
Good catch! I'll try finding such an ESLint rule. Thanks.

Reply:
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md

Reply:
Thanks. I've opened PR #6942.

Reply:
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/group-exports.md would be useful too